src/java.base/share/classes/java/lang/StringBuilder.java

```diff
@@ -90,7 +90,7 @@
  */
 public final class StringBuilder
     extends AbstractStringBuilder
-    implements java.io.Serializable, Comparable<StringBuilder>, CharSequence
+    implements Appendable, java.io.Serializable, Comparable<StringBuilder>, CharSequence
```

Comment:
Not an issue here, just wondering why do we have lots of places where we use `java.io.Serializable` instead of import statement?

Reply:
The JDK codebase historically has not used tooling to enforce a uniform style on such matters.